### PR TITLE
Sigstore repo can now be listed and confirmed

### PR DESCRIPTION
**Pull request by breillyr**

### CODE REVIEW

File: dockpulp/__init__.py [hunk 1]
```diff
@@ -161,6 +162,17 @@ def confirm(self, repos, v1=True, v2=True, silent=True, check_layers=False):
         for repo in repos:
             log.info('Testing %s' % repo['id'])
             repoids[repo['id']] = {}
+            errorids[repo['id']] = False
+            if repo['id'] == SIGSTORE:
+                errorids[repo['id']] = False
```
Comment (twaugh) [inline dockpulp/__init__.py]: This line isn't needed.

File: dockpulp/__init__.py [hunk 2]
```diff
@@ -589,6 +600,26 @@ def _test_repoV2(self, dockerid, redirect, pulp_manifests, pulp_blobs, pulp_tags
 
         return result
 
+    def _test_sigstore(self, signatures):
+        """Confirm we can reach CDN and get data back from it."""
+        result = {}
+        result['error'] = False
+        result['missing_signatures'] = []
```
Comment (twaugh) [inline dockpulp/__init__.py]: Could be written as:
```
result = {'error': False, 'missing_signatures': []}
```

File: dockpulp/__init__.py [hunk 3]
```diff
@@ -1319,6 +1354,14 @@ def listRepos(self, repos=None, content=False, history=False, labels=False):
                 units = self._post(
                     '/pulp/api/v2/repositories/%s/search/units/' % blob['id'],
                     data=json.dumps(data))
+                if blob['id'] == SIGSTORE:
+                    r['sigstore'] = []
+                    sigs = [unit for unit in units
+                            if unit['unit_type_id'] == SIG_TYPE]
+                    for sig in sigs:
+                        r['sigstore'].append(sig['metadata']['name'])
+                    clean.append(r)
```
Comment (twaugh) [inline dockpulp/__init__.py]: Other places using this also sort it.

Comment (twaugh) [inline dockpulp/__init__.py]: I don't think this section gets covered in unit testing.

**Comment by breillyr** [comment]
Section is now covered by unit testing, going to go ahead and merge

**Pull request merged**